app/views/draftmetadata/draftMetadataUpload.scala.html

```diff
@@ -5,11 +5,35 @@
 @import java.util.UUID
 @(consignmentId: UUID, consignmentRef: String, frontEndInfo: FrontEndInfo, name: String)(implicit request: RequestHeader, messages: Messages)
 
-@main("[WIP] Draft Metadata Upload", name = name) {
-    <div id="draft-metadata-upload" class="govuk-grid-row">
+@main("Draft Metadata Upload", name = name, backLink = Some(backLink(routes.AdditionalMetadataEntryMethodController.additionalMetadataEntryMethodPage(consignmentId).url, "How would you like to enter metadata?"))) {
```

Comment:
The "main" title should match the title in the `h1` element: ``Upload a metadata CSV`